npm-debug.log

```diff
@@ -0,0 +1,168 @@
+0 info it worked if it ends with ok
```

Comment:
You probably don't want to synchronize the npm-debug.log It may contain your personal information.